app/src/test/java/org/simple/clinic/analytics/AnalyticsTest.kt

```diff
@@ -167,8 +190,13 @@ class AnalyticsTest {
     val userId = UUID.randomUUID()
```

Comment:
Shouldn't the name of the test say "...user id must ~by~ **be** set on..."?

Reply:
Whoops, will fix/